core/field_textinput.js

```diff
@@ -46,6 +46,9 @@ goog.require('goog.userAgent');
  * @constructor
  */
 Blockly.FieldTextInput = function(text, opt_validator) {
+  if (!text) {
```

Comment:
Same issue, and empty string should be a valid starting point for text input fields.

Reply:
Fixed